models/api.articles.comments.model.js

```diff
@@ -22,3 +22,14 @@ exports.fetchArticleById = (article_id) => {
     }
   });
 };
+
+exports.insertComment = (article_id, username, body) => {
+  const sqlQuery = `INSERT INTO comments (body, votes, author, article_id, created_at) 
+  VALUES ($1, $2, $3, $4, $5)
+  RETURNING *;`
+  const queryValues = [body, 0, username, article_id, new Date]
```

Comment:
The seed has a default for the created_at - so you don't need to add it here. It will be generated automatically!